src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24061.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue24061 : _IssuesUITest
+	{
+		public Issue24061(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "ScrollView's gradient background doesn't work with ScrollView";
+
+		[Test]
+		[Category(UITestCategories.ScrollView)]
+		public void ScrollViewGradientBackgroundShouldWorkWithScrollView()
+		{
+			App.WaitForElement("Item2");
+			App.SwipeRightToLeft("Item2", 0.99);
+
+			VerifyScreenshot("ScrollViewGradientBackground");
+			App.Click("button");
+			VerifyScreenshot("ScrollViewWithoutGradientBackground");
+			App.Click("button");
+			VerifyScreenshot("ScrollViewGradientBackground");
```

Comment:
There are pending snapshots. Are available on the latest build.
![image](https://github.com/user-attachments/assets/1d1bac59-00ea-4db1-96d8-76705751288f)

Could you commit them?

Reply:
Could you please /azp again? I need one more screenshot :)